frontend/src/components/Pages/MissionPage/VideoStream/VideoPlayerOvenPlayer.tsx

```diff
@@ -41,7 +41,7 @@ export function VideoPlayerOvenPlayer({ videoStream }: IVideoPlayerProps) {
     return <div id={videoStream.id} />
 }
 
-export function IsValidOvenPlayerType({ videoStream }: IVideoPlayerProps) {
+export const isValidOvenPlayerType = ({ videoStream }: IVideoPlayerProps) => {
```

Comment:
Since we only have one argument here, we could probably jsut have videoStream be a normal parameter instead of in an object.